### PR TITLE
Updates to copy for rebranding

### PR DESCRIPTION
**Pull request by osahyoun**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -41,10 +41,10 @@ jobs:
               aws s3 sync build s3://staging.homepage.sumofus.org --acl public-read --cache-control "public, max-age=0" --delete --region us-west-2
             fi
             if [ "${CIRCLE_BRANCH}" == "master" ]; then
-              aws s3 sync build s3://www.sumofus.org --acl public-read --cache-control "public, max-age=0" --delete --region us-east-1
```
Comment (yeseniamolinab) [inline .circleci/config.yml:44]: @osahyoun Don't we need to change the s3 bucket first?

File: config.rb [hunk 2]
```diff
@@ -25,8 +25,8 @@
 
 # Build-specific configuration
 configure :build do
-  config[:api_host] = "https://actions.sumofus.org"
-  # config[:api_host] = "https://action-staging.sumofus.org"
+  config[:api_host] = "https://actions.eko.org"
```
Comment (yeseniamolinab) [inline config.rb:28]: We need to create the actions.eko.org subdomain (unless that's already done?)

**Review by yeseniamolinab — APPROVED** [review]
@osahyoun LGTM,  just have two small comments

File: .circleci/config.yml (hunk 1 above)
Comment (osahyoun) [reply .circleci/config.yml:44]: Indeed! I'll create a new bucket for this.

**Pull request merged**